core/common/src/main/java/alluxio/collections/IndexedSet.java

```diff
@@ -27,8 +27,8 @@
 
 /**
  * A set of objects that are indexed and thus can be queried by specific fields of the object.
- * Different {@link IndexedSet} instances may specify different fields to index. The field type
- * must be comparable. The field value must not be changed after an object is added to the set,
+ * Different {@link IndexedSet} instances may specify different fields to index. The field type must
```

Comment:
Sorry for the inconvenience. Those formatting changes were caused by the formatter plugin, I'll be careful next time.